### PR TITLE
'Formatted  table heading and added dasboard nav bar'

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Separated table heading and body by formatting the colours on both.
I also added a navigation bar and wrapped the table in a container element.
dashboard

### CODE REVIEW

**Comment by lukechurch** [comment]
Thanks @IsaackMwenda, per discussion I'll let Rita review first.

For things like this, it's pretty difficult to imagine what it looks like - could you also take a quick screenshot and embed it in the PR comment?

**Comment by IsaackMwenda** [comment]
> Thanks @IsaackMwenda, per discussion I'll let Rita review first.
> 
> For things like this, it's pretty difficult to imagine what it looks like - could you also take a quick screenshot and embed it in the PR comment?
@lukechurch Great have added a screenshot.

File: public/coda/coding_progress.html
Comment (lukechurch) [inline public/coda/coding_progress.html]: This looks a bit odd. Could you elaborate on what you were aiming to achieve with the <a> tag referencing an image as its target?

File: public/coda/coding_progress.html
Comment (lukechurch) [inline public/coda/coding_progress.html]: Please could you indent this as it's enclosed within the div

File: public/coda/coding_progress.html
Comment (lukechurch) [inline public/coda/coding_progress.html]: Nice :)

File: public/coda/coding_progress.js
Comment (lukechurch) [inline public/coda/coding_progress.js]: I don't think there's a need for this additional whitespace? it doesn't really help reading and results in less code on the screen

File: public/coda/styles.css
Comment (lukechurch) [inline public/coda/styles.css]: New line at the end of the file please, so that if you 'cat' the code into a terminal, it ends nicely

**Review by lukechurch — COMMENTED** [review]
A few tweaks to go before we land this please, nothing major though and a good step in the right direction visually

File: public/coda/coding_progress.html
Comment (IsaackMwenda) [reply public/coda/coding_progress.html]: I wanted to include the AVF logo with the dashboard title. Have fixed it 
image

File: public/coda/coding_progress.html
Comment (IsaackMwenda) [reply public/coda/coding_progress.html]: Noted

File: public/coda/coding_progress.js
Comment (IsaackMwenda) [reply public/coda/coding_progress.js]: Great have removed it

**Comment by lukechurch** [comment]
Hi @IsaackMwenda - after you've made the changes locally, you need to commit them to your local copy of the branch, and then push that branch to Github. That way I'll be able to see the changes you've made, and review again

File: public/coda/styles.css
Comment (ritazagoni) [inline public/coda/styles.css]: Looks like this should have the same indentation as the others as on the same level?

**Review by ritazagoni — APPROVED** [review]
Thanks @IsaackMwenda one minor comment. Once that's fixed and pushed pls request final review from Luke.

File: public/coda/styles.css
Comment (IsaackMwenda) [reply public/coda/styles.css]: Noted. Thank you

**Comment by IsaackMwenda** [comment]
@lukechurch Have fixed indentation, removed white spaces and added a new line at the end of each file. Kindly review.

**Review by lukechurch — APPROVED** [review]
Looks good to me! Thanks @IsaackMwenda 

Please merge and push to firebase hosting

**Pull request closed without merging**
